beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -5798,15 +5799,30 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 let kzg_proofs = Vec::from(proofs);
 
                 let kzg = self.kzg.as_ref();
-
-                // TODO(fulu): we no longer need blob proofs from PeerDAS and could avoid computing.
-                kzg_utils::validate_blobs::<T::EthSpec>(
-                    kzg,
-                    expected_kzg_commitments,
-                    blobs.iter().collect(),
-                    &kzg_proofs,
-                )
-                .map_err(BlockProductionError::KzgError)?;
+                if self
+                    .spec
+                    .is_peer_das_enabled_for_epoch(slot.epoch(T::EthSpec::slots_per_epoch()))
+                {
+                    // TODO: extend blobs and verify cells against proofs
```

Comment:
```suggestion
                    // TODO(das): extend blobs and verify cells against proofs
```

Reply:
Implemented in https://github.com/sigp/lighthouse/commit/7f65c506fc963c89199ef3458dedd58f4cdcb0b9